packages/editor/src/useMedia.ts

```diff
@@ -0,0 +1,70 @@
+import { useCallback, useEffect } from 'react';
```

Comment:
Might want to put this in the shared package, we can pull it into the new web app later too.

Not a blocker necessarily.